README.md

```diff
@@ -4,11 +4,36 @@ Notation is a project to add signatures as standard items in the registry ecosys
 
 ## Table of Contents
 
-1. [Prototype 2][prototype-2] - signing and verifying OCI artifacts, using signatures persisted [ORAS Artifacts manifests][artifact-manifest]
+- [Branch](#branch)
+- [Core Documents](#core-documents)
+- [Community](#community)
+- [Code of Conduct](#code-of-conduct)
+- [License](#license)
+
+## Branch
+
+[Prototype 2][prototype-2] - signing and verifying OCI artifacts, using signatures persisted [ORAS Artifacts manifests][artifact-manifest]
 
 [artifact-manifest]:  https://github.com/oras-project/artifacts-spec/blob/main/artifact-manifest.md
 [prototype-2]:      https://github.com/notaryproject/notation/tree/prototype-2
 
+## Core Documents
+
+* [Governance for Notation](https://github.com/notaryproject/notary/blob/master/GOVERNANCE.md)
+* [Maintainers and reviewers list](https://github.com/notaryproject/notary/blob/master/MAINTAINERS)
+
+## Community
+
+* Regular conversations for Notation occur on the [Cloud Native Computing Slack](https://app.slack.com/client/T08PSQ7BQ/CQUH8U287?) channel.
+
+* Weekly meetings occur each Monday. Please see the [CNCF Calendar](https://www.cncf.io/community/calendar/) for details.
```

Comment:
Can we avoid stating a date, and just point to the [CNCF calendar](https://www.cncf.io/calendar/) so we don't have to track changes in multiple locations?

Reply:
Sounds good!